### PR TITLE
fix: [DO NOT MERGE] Rfq registry

### PR DESCRIPTION
**Pull request by PirosB3**
- Adds a new `JsonStringList` formatter
- Adds a new environment variable called `RFQT_REGISTRY_PASSWORDS` which should contain the authorization tokens that our market makers can use to redeem the 0x API integrator registry API
- Adds the 0x API integrator registry API endpoint
- Changes `RFQT_API_KEY_WHITELIST` and `RFQT_API_KEY_WHITELIST` to read from their JSON equivalent (This PR is to be used with Infra PR #1048

### CODE REVIEW

File: .env_example [hunk 1]
```diff
@@ -26,8 +26,7 @@ META_TXN_RELAY_PRIVATE_KEYS=
 META_TXN_RELAY_EXPECTED_MINED_SEC=
 ENABLE_PROMETHEUS_METRICS=
 PROMETHEUS_PORT=
-RFQT_API_KEY_WHITELIST=
-PLP_API_KEY_WHITELIST=*
+RFQT_API_KEY_WHITELIST_JSON=
```
Comment (alexkroeger) [inline .env_example:29]: Shouldn't there be `PLP_API_KEY_WHITELIST_JSON` as well?

File: src/handlers/swap_handlers.ts [hunk 2]
```diff
@@ -45,12 +46,39 @@ import {
 
 import { quoteReportUtils } from './../utils/quote_report_utils';
 
+const BEARER_RE = /^Bearer\s(.{36})$/;
```
Comment (alexkroeger) [inline src/handlers/swap_handlers.ts:49]: Optional nit: rename to `BEARER_REGEX`

File: src/handlers/swap_handlers.ts [hunk 3]
```diff
@@ -45,12 +46,39 @@ import {
...
         const message = `This is the root of the Swap API. Visit ${SWAP_DOCS_URL} for details about this API.`;
         res.status(HttpStatus.OK).send({ message });
     }
+
+    public static async getRfqRegistryAsync(req: express.Request, res: express.Response): Promise<void> {
+        const auth = req.header('Authorization');
+        if (auth === undefined) {
+            return res.status(HttpStatus.UNAUTHORIZED).end();
+        }
+        REGISTRY_ENDPOINT_FETCHED.labels(auth).inc();
```
Comment (alexkroeger) [inline src/handlers/swap_handlers.ts:69]: Maybe place the counter ahead of auth === undefined if we want to see all requests?

**Review by alexkroeger — CHANGES_REQUESTED** [review]
LGTM, check nit in .env_example and I'll approve

File: .env_example (hunk 1 above)
Comment (PirosB3) [reply .env_example:29]: Good call, will add!

File: src/handlers/swap_handlers.ts (hunk 3 above)
Comment (PirosB3) [reply src/handlers/swap_handlers.ts:69]: good call!

**Review by alexkroeger — APPROVED** [review]

**Comment by kimpers** [comment]
deploy staging

**Pull request merged**